libs/velo-external-db-core/src/router.ts

```diff
@@ -181,36 +171,31 @@ export const createRouter = () => {
                 filterTransformer.transform(filter),
             )
 
-            const dataAfterAction = await executeDataHooksFor(DataActions.AfterCount, data, requestContextFor(Count, req.body, res.locals), customContext)
-
-            const response = {
-                totalCount: dataAfterAction.totalCount
-            } as dataSource.CountResponse
+            const { totalCount } = await executeDataHooksFor(DataActions.AfterCount, data, requestContextFor(Count, req.body, res.locals), customContext)
 
-            res.json(response)
+            res.json({ totalCount })
         } catch (e) {
             next(e)
         }
     })
 
+    // TODO: implement error structure (ApplicationError type)
```

Comment:
?

Reply:
My mistake I made at the beginning of working on this PR